packages/dfw/src/dfwcomponents/WorkOrderCell/PropertyList.tsx

```diff
@@ -18,34 +17,24 @@ const defaultWorkOrderLabelMap: Record<keyof WorkOrder, string> = {
 } as const;
 
 type PropertyListProps = {
-    workOrder: Partial<WorkOrder>;
+    data: Partial<WorkOrder>;
     valueColor: Color;
-    currentDate: Date;
-    overwriteLabel?: Partial<Record<keyof WorkOrder, string>>;
 };
 
-export const PropertyList = ({
-    workOrder,
-    valueColor,
-    currentDate,
-    overwriteLabel = {},
-}: PropertyListProps) => {
-    const token = useToken();
-
-    const formatDate = (dateString: string) => moment(dateString).format("DD.MM.YYYY");
-
+export const PropertyList = ({ data: workOrder, valueColor }: PropertyListProps) => {
```

Comment:
rename-refactor seems to have still kept the name of this field `workOrder` internally in this component

Reply:
ops